AmplifyPlugins/API/AWSAPICategoryPluginTests/Operation/OperationTestBase.swift

```diff
@@ -12,21 +12,23 @@ import XCTest
 
 class OperationTestBase: XCTestCase {
 
+    var apiPlugin: AWSAPIPlugin!
+
     /// There are no throwing methods in this, but in order to let subclasses have a predictable way
     /// of setting up plugins, this base class uses `setUpWithError` instead of `setUp`. (XCTest
     /// lifecycle normally invokes `setUp` after `setUpWithError`, which means that any state config
     /// done inside of a subclass' `setUpWithError` could be erased by this class' call to
     /// `Amplify.reset` from inside `setUp`.
     override func setUpWithError() throws {
-        Amplify.reset()
+        apiPlugin = nil
```

Comment:
Does `AWSAPIPlugin.reset()` do anything to cancel in-process operations? I'm not sure whether just setting `nil` will do enough to clear in-process/shared state

Reply:
Added reset for plugin